geode-core/src/main/java/org/apache/geode/management/internal/cli/domain/DataCommandRequest.java

```diff
@@ -195,35 +195,35 @@ public void setPrincipal(Object principal) {
     this.principal = principal;
   }
 
-  // @Override
-  public void toData(DataOutput out) throws IOException {
-    DataSerializer.writeString(command, out);
-    DataSerializer.writeString(key, out);
-    DataSerializer.writeString(value, out);
-    DataSerializer.writeBoolean(putIfAbsent, out);
-    DataSerializer.writeString(keyClass, out);
-    DataSerializer.writeString(valueClass, out);
-    DataSerializer.writeString(regionName, out);
-    DataSerializer.writeString(removeAllKeys, out);
-    DataSerializer.writeBoolean(recursive, out);
-    DataSerializer.writeBoolean(loadOnCacheMiss, out);
-    DataSerializer.writeObject(principal, out);
-  }
-
-  // @Override
-  public void fromData(DataInput in) throws IOException, ClassNotFoundException {
-    command = DataSerializer.readString(in);
-    key = DataSerializer.readString(in);
-    value = DataSerializer.readString(in);
-    putIfAbsent = DataSerializer.readBoolean(in);
-    keyClass = DataSerializer.readString(in);
-    valueClass = DataSerializer.readString(in);
-    regionName = DataSerializer.readString(in);
-    removeAllKeys = DataSerializer.readString(in);
-    recursive = DataSerializer.readBoolean(in);
-    loadOnCacheMiss = DataSerializer.readBoolean(in);
-    principal = DataSerializer.readObject(in);
-  }
+  // // @Override
```

Comment:
Why is this commented out?

Reply:
Looks like the class author had trouble making this class implement DataSerializable, so they changed it to:
```java
public class DataCommandRequest implements /*Data*/ Serializable{
```
And then commented out the `@Override` annotations on toData and fromData methods.

I'd recommend just deleting toData and fromData methods entirely since they are dead code.

Reply:
This class was in the openBugs.txt file & the associated ticket was closed two years ago so I removed it.  When I did that the testDataSerializables unit test objected that this class had fromData/toData methods but it isn't DataSerializable so those methods will never be used.  Since there was a TODO in this class to make it DataSerializable I didn't feel comfortable just deleting the methods so I commented them out.

Reply:
It seems like a Jira ticket might be a better place for this rather than keeping a TODO with commented out code in the class itself.

Reply:
It's not my class and I didn't add the TODO.  I think it's a gfsh class.  I can remove the TODO and commented out code if you like.

Reply:
Deleting the TODO and commented out code sounds good.